src/widget/color.ts

```diff
@@ -23,22 +23,35 @@ import { vec3 } from "#src/util/geom.js";
 export class ColorWidget<
   Color extends vec3 | undefined = vec3,
 > extends RefCounted {
-  element = document.createElement("input");
+  static template() {
```

Comment:
I think the changes in this file can be reverted now.

Reply:
oops... done!